qa/rpc-tests/test_framework/util.py

```diff
@@ -548,7 +548,4 @@ def create_lots_of_big_transactions(node, txouts, utxos, fee):
 
 def get_bip9_status(node, key):
     info = node.getblockchaininfo()
-    for row in info['bip9_softforks']:
-        if row['id'] == key:
-            return row
-    raise IndexError ('key:"%s" not found' % key)
+    return info['bip9_softforks'][key]
```

Comment:
Oh, actually this is not the issue of the code here. To get a better error message, you'd have to change `str` into `repr` in this line: https://github.com/bitcoin/bitcoin/blob/514993554c370f4cf30a109ac28d5d64893dbf0a/qa/rpc-tests/test_framework/test_framework.py#L146